test/unit/migrations/migrator-test.js

```diff
@@ -35,6 +37,21 @@ const firstTimeState = {
   meta: { version: 0 },
   data: require('../../../app/scripts/first-time-state'),
 }
+describe('liveMigrations require list', () => {
+  it('should include all the migrations', async () => {
+    const fileNames = await pify(cb => fs.readdir('./app/scripts/migrations/', cb))()
+    const migrationNumbers = fileNames.reduce((agg, filename) => {
+      const name = filename.split('.')[0]
+      if (/^\d+$/.test(name)) agg.push(name)
```

Comment:
Whoops, this was prevented by a lint rule since this was first written

Reply:
gahhh okie